client/asset/btc/rpcclient.go

```diff
@@ -658,7 +658,7 @@ func (wc *rpcClient) call(method string, args anylist, thing interface{}) error
 }
 
 // serializeMsgTx serializes the wire.MsgTx.
-func serializeMsgTx(msgTx *wire.MsgTx) ([]byte, error) {
+func serializeMsgTx(msgTx *wire.MsgTx) (dex.Bytes, error) {
```

Comment:
Revert this back to `[]byte` please.  I want to be using native types more, `dex.Bytes` less.  Conversion between them is automatic in most cases.  The only reason we need to ever use `dex.Bytes` is when we need to rely on it's `String`, `MarshalJSON`, or `UnmarshalJSON` methods (such as when the `btc.GetTransactionResult.Hex dex.Bytes` field is marshalled/unmarshalled. Otherwise we're just obscuring things.  Arguably the `Hex` field in `GetTransactionResult` is confusing for this reason, and we have to have a comment explaining it and hopefully the it's marshal/unmarshal methods never break and wreck things.  If we use `dex.Bytes` we should have a good reason and document it.